src/main/resources/doc/en/html/guide/tutorial/tutor-gates.html

```diff
@@ -46,7 +46,7 @@ <h2>
         I left a little space between the NOT gates and the AND gates; if you want to, though, you can put them up against each other and save yourself the effort of connecting them with a wire later.
       </p>
       <p>
-        Now we want to add the two inputs <var>x</var> and <var>y</var> into the diagram. Select the <b class=propertie>"Input"</b> tool (<img class="icontxt" src="../../../../icons/1616/toolpininput.png" alt="#########">), and place the pins down. You should also place an output pin next to the OR gate's output using the <b class=propertie>"Output"</b> tool (<img class="icontxt" src=
+        Now we want to add the two inputs <var>x</var> and <var>y</var> into the diagram. Select the <b class=propertie>"Input"</b> tool (<img class="icontxt" src="../../../../icons/1616/toolpinintput.png" alt="#########">), and place the pins down. You should also place an output pin next to the OR gate's output using the <b class=propertie>"Output"</b> tool (<img class="icontxt" src=
```

Comment:
I do not think this is correct way to fix that. I'd rather rename the image (and fix other references to badly spelled file name).

Reply:
Fixed in 240601c1fd493aacb83ce41a22c898cd4f0d9bb8